src/CMakeLists.txt

```diff
@@ -102,6 +102,16 @@ if ( NOT HAVE_GETOPT_LONG )
     target_link_libraries(zeek-agent PRIVATE 3rdparty:3rdparty)
 endif()
 
+if ( WIN32 )
+    # Windows requires administrator rights for various things (especially getting process names in the socket module)
+#    set_target_properties(zeek-agent PROPERTIES LINK_FLAGS "/MANIFESTUAC:\"level='requireAdministrator' uiAccess='false'\" /SUBSYSTEM:WINDOWS")
```

Comment:
I had it off during testing because it's really annoying to have to start up Visual Studio in Administrator mode. I'll enable it before I merge the PR.

Reply:
So sounds like we don't want to activate this then, so that it can also run without admin?